src/index.html

```diff
@@ -4,9 +4,459 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page">
+    <header class="header">
+      <a class="header__home-link"
+        href="#Home">
```

Comment:
```suggestion
      <a
        class="header__home-link"
        href="#Home"
      >
```

Keep all attributes on the same level.

Reply:
fixed